### ISSUE TITLE
embedding a photo from the repo

### ISSUE THREAD

**Issue by nbrown20** [cfss-fa20/hw01#18]
hi! I am trying to embed a photo in my markdown file from my repo but I keep getting an error. I've tired multiple paths but I basically keep getting "File lou.jpeg not found in resource path". What should I try next?

**Comment by djanvier** [comment]
Hello. Would you be able to copy and paste the code you used? I see the image in your repo, so you should be able to use a relative file path.

**Comment by nbrown20** [comment]
sure, thanks!

i tried ![](lou.jpeg)
and ![](nbrown20/hw01/lou.jpeg)
and ![](hw01/lou.jpeg)

is the code showing? it's not in the preview of my message -- ill attach an image

<img width="283" alt="Screen Shot 2020-10-03 at 8 32 36 PM" src="https://user-images.githubusercontent.com/72172521/95004938-94e32400-05b7-11eb-86fd-83eefa148a7a.png">

**Comment by djanvier** [comment]
The first line of code you tried `![](lou.jpeg)` looks correct. I'm not sure why it's not working. Could you push your most recent changes to GitHub?
Also, you can insert code in your comments using two backticks `` or include a reproducible example using the [`reprex`](https://cfss.uchicago.edu/faq/asking-questions/#format-your-code-snippets-with-reprex) package

**Comment by nbrown20** [comment]
sure, i'll try -- thanks!

**Comment by nbrown20** [comment]
<img width="611" alt="Screen Shot 2020-10-03 at 9 10 28 PM" src="https://user-images.githubusercontent.com/72172521/95005352-e0e49780-05bc-11eb-8a60-be272c2278c6.png">


this is the message I get

**Comment by djanvier** [comment]
Oh, I see now! So this error means that some changes were made on GitHub that were not reflected locally. My guess is that it's referring to the photo you added on GitHub. In order for the repos to be the same, you need to **pull** from the repository on GitHub and  _then_ push your changes. If you pull correctly, the photo should be in the same folder as the README.md in your local version of R Studio. Let me know if you have any trouble with this!

**Comment by nbrown20** [comment]
thank you! do I have to create a new project to do this after I copy the code? thanks for your help so far!

**Comment by djanvier** [comment]
Nope! You should just be able to pull the changes from GitHub to your local repository.

**Comment by nbrown20** [comment]
It is not allowing me to pull the changes? Is git pull not the right command?

**Comment by deblnia** [comment]
What’s the error message you’re seeing when you run git pull?

**Comment by nbrown20** [comment]
Error: unexpected symbol in "git pull"

**Comment by nbrown20** [comment]
thanks for all this help :)

**Comment by deblnia** [comment]
Are you using the R console or the terminal tab? If that doesn’t mean much to you, try the git tab (to the right of the environment pane). You can click pull there.

**Comment by nbrown20** [comment]
ohhh I see -- thank you so much!
